### ISSUE TITLE
Figured it out.

### ISSUE THREAD

**Issue by luukipuuk1912** [Tune389/Tap-Titans-2-Progress-Bot#62]
After horribly failing at my first attempt of using the script, i got it under control.
I've noticed it forgets to level skills after prestige, I managed to somehow fix it, by just letting it open the menu and sliding up all the way to swordmaster. I have designed it to my liking, adjusting random tap and skills and sorts. so you'll have to change that. (If you even want the file in the first place.)
thanks for your awesome work! I'll check for a response and if wanted a copy of my file :)

**Comment by rockfreak101** [comment]
yeah, lets see it. I'm having quite a few issues today after the update.

**Comment by Derek1991** [comment]
@luukipuuk1912 Are you able to link your entire script?
